packages/seo-store/src/editor/slice.js

```diff
@@ -35,9 +35,10 @@ const editorSlice = createSlice( {
 			state.featuredImage = action.payload;
 		},
 		updateTerms: ( state, action ) => {
-			const taxonomyType = action.payload.taxonomyType;
 			const terms = action.payload.terms;
-			state.taxonomies[ taxonomyType ] = terms;
+			const taxonomyType = action.payload.taxonomyType;
+
+			set( state, "taxonomies." + taxonomyType, terms );
```

Comment:
To the reviewers:

I changed the way we update the terms to use the `set` method from lodash here, so that it's possible to chain a key in the `taxonomyType` that is being passed. For example "customTaxonomies.actors".